jenkins/release-manifest-commit-lock/release-manifest-commit-lock.jenkinsfile

```diff
@@ -137,6 +141,41 @@ pipeline {
                 }
             }
         }
+        stage('UPDATE_TO_TAGS') {
+            when {
+                expression { params.MANIFEST_LOCK_ACTION == 'UPDATE_TO_TAGS' }
+            }
+            steps {
+                script {
+                    def updateManifest = { String productName ->
+                        def existingManifest = readYaml file: "manifests/${params.RELEASE_VERSION}/${productName}-${params.RELEASE_VERSION}.yml"
+                        def selectedComponents = params.COMPONENTS.isEmpty() ? existingManifest.components : existingManifest.components.findAll { component ->
+                            params.COMPONENTS.split(',').any { it.trim() == component.name }
+                        }
+                        selectedComponents.each { componentName ->
+                            def existingComponent = existingManifest.components.find { it.name == componentName.name }
+                            if (existingComponent) {
+                                def releaseTag = params.RELEASE_VERSION + ".0"
+                                if (componentName.name == 'OpenSearch' || componentName.name == 'OpenSearch-Dashboards' || componentName.name == 'functionalTestDashboards') {
+                                    releaseTag = params.RELEASE_VERSION
+                                }
+                                def newComponentRef = existingManifest.components.find { it.name == componentName.name }?.ref
+                                        if (newComponentRef) {
+                                            existingComponent.ref = "tags/" + releaseTag
+                                        }
+                            }
```

Comment:
Why do we need to do this? We should just retrieve the tag directly right? Instead of manually adding a 0 here.

Reply:
Got it.